utils/utils.jsx

```diff
@@ -634,7 +634,7 @@ export function applyTheme(theme) {
         changeCss('.app__body .shortcut-key, .app__body .post-list__new-messages-below', 'color:' + theme.centerChannelBg);
         changeCss('.app__body .emoji-picker, .app__body .emoji-picker__search', 'background:' + theme.centerChannelBg);
         changeCss('.app__body .nav-tabs, .app__body .nav-tabs > li.active > a', 'background:' + theme.centerChannelBg);
-        changeCss('.app__body .modal-tabs .nav-tabs > li', `background:${theme.centerChannelBg}`);
+        changeCss('.app__body .modal-tabs .nav-tabs > li', 'background:' + theme.centerChannelBg);
```

Comment:
@d-schaffer can you remove this line? We are replacing the function call with the CSS variables.

Reply:
@d-schaffer do you have questions about this feedback?